script/cibuild

```diff
@@ -7,3 +7,19 @@ bundle exec htmlproofer ./_site --check-html --check-sri
 bundle exec rubocop -D
 bundle exec script/validate-html
 gem build jekyll-theme-primer-spec.gemspec
+
+# Sanity check that JavaScript was built properly and committed.
+rm -rf tmp/
+mkdir -p tmp/
+git clean -xdf src_js/ assets/
+git checkout -- assets/js/spec_main.js
```

Comment:
Just checking if you're sure you want an automated script to issue Git commands.  What if the developer had modified `spec_main.js`?  This would clobber their changes.  Would check-and-fail be a viable alternative here?

Reply:
Good point. I'll modify the script to run this check only on Travis.